beacon-chain/blockchain/service.go

```diff
@@ -217,6 +199,23 @@ func (c *ChainService) updateHead(slot uint64) {
 	// server to stream these events to beacon clients.
 	transition := c.chain.IsCycleTransition(slot)
 	if transition {
+		if err := c.chain.SetCrystallizedState(c.candidateCrystallizedState); err != nil {
+			log.Errorf("Write crystallized state to disk failed: %v", err)
+		}
+
+		// TODO: Utilize this value in the fork choice rule.
```

Comment:
Please reference a github issue. The TODO should be in the format TODO(1234) where 1234 is the issue number.

I'm not sure if there is a fork choice rule issue. @rauljordan  can you confirm?